board/app/controllers/favorite_controller.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class FavoriteController extends AppController 
+{
+
+ public function set_favorite()
```

Comment:
Please indent this line (function) since this is inside the class FavoriteController :bow: